tasks/other/install-deps.sh

```diff
@@ -6,11 +6,11 @@ npm run build
 composer install
 popd || exit
 
-for plugin in gutenberg-blocks gutenberg-engagingnetworks
+for plugin in gutenberg-blocks
```

Comment:
Since we now have only on repo here, maybe we can simplify the code and just drop the for loop completely.

Reply:
Done. Please note: I haven't tested it locally (as it would include modifying the whole planet4-docker-compose to pick things from a different repo)